tcl/board/gemini.cfg

```diff
@@ -20,7 +20,7 @@ jtag newtap $_CHIPNAME tap -irlen 5 -expected-id 0x1000563d
 target create $_TARGETNAME riscv -endian little -chain-position $_CHIPNAME.tap
 
 # Gemini PLD/FPGA configuration
-pld device gemini $_TARGETNAME
+pld create gemini.pld gemini $_TARGETNAME
```

Comment:
We will need update the fpga configuration command in the programmer tool

Reply:
Yes.